src/cpu/x86/lapic/lapic_cpu_stop.c

```diff
@@ -55,6 +55,8 @@ void stop_this_cpu(void)
 
 	printk(BIOS_DEBUG, "CPU %ld going down...\n", id);
 
+	atomic_inc(&parked_ap_count);
```

Comment:
This needs a comment explaining why the increment is done in this particular place and not above or below.

Reply:
Added a comment